sdk/helper/logging/logging.go

```diff
@@ -60,16 +60,13 @@ func ParseLogFormat(format string) (LogFormat, error) {
 	case "json":
 		return JSONFormat, nil
 	default:
-		return UnspecifiedFormat, fmt.Errorf("Unknown log format: %s", format)
+		return UnspecifiedFormat, fmt.Errorf("unknown log format: %s", format)
 	}
 }
 
 // ParseEnvLogFormat parses the log format from an environment variable.
 func ParseEnvLogFormat() LogFormat {
```

Comment:
The one thing that's not immediately clear to me, and probably doesn't need to be answered in this PR, is why is this function necessary?

If we use `EnvVar` in the `StringVar` in the command side, do we need to call this as well/alternatively? Or can we just rely on the option variable handling this instead?